qutebrowser/qt/_core_pyqtproperty.py

```diff
@@ -9,7 +9,7 @@
 """
 
 # flake8: noqa
-# pylint: disable=invalid-name,missing-class-docstring,too-many-arguments,redefined-builtin,unused-argument
+# pylint: disable=invalid-name,missing-class-docstring,too-many-arguments,redefined-builtin,unused-argument,deprecated-typing-alias
```

Comment:
Didn't change the `typing` usage in here as this was a 1:1 backport from PyQt5-stubs I think.